features/step_definitions/kata.rb

```diff
@@ -227,3 +227,78 @@
     end
   end
 end
+
+Given /^Verify catalog source existence$/ do
+  project_name = 'openshift-marketplace'
+  catalog_source_name = 'redhat-operators'
+  step %Q/I switch to cluster admin pseudo user/
+  project(project_name)
+  if !catalog_source(catalog_source_name).exists?
+    logger.warn("Missing #{catalog_source_name} catalog source")
+    logger.info("Creating #{catalog_source_name}")
+    step %Q/I create "redhat-operators" catalogsource for testing/
```

Comment:
@valtair1891 where is this step defined?  I can't seem to find it in this PR

Reply:
It's used in line 127, i can find the exact location if you want